packages/fast-components-react-msft/src/carousel/carousel.tsx

```diff
@@ -22,6 +22,12 @@ export interface CarouselState {
     activeId: string;
 }
 
+enum SlideTransitionDirection {
+    previous = "previous",
+    next = "next",
+    initial = "",
```

Comment:
Curious if initial to null or maybe if this might be an index and then set to 0 as default. That way it's possible to set to any slide as the start.  Just making stuff up on my end.  But, initial isn't really in the language i've used for slides before.

Reply:
When a carousel loads fresh, it has no sense of "previous slide direction" because nothing has slid. I am simply setting the starting variable to an empty string in a slightly more fancy way.

I could just do 
`private slideTransitionDirection: Partial<SlideTransitionDirection> = "";`
instead of
`private slideTransitionDirection: Partial<SlideTransitionDirection> = SlideTransitionDirection.initial;`

Reply:
since the enum isn't exported and is just internal, can we simply remove it?